run_pacta_data_preparation.R

```diff
@@ -1,21 +1,19 @@
+logger::log_threshold(Sys.getenv("LOG_LEVEL", "INFO"))
+logger::log_formatter(logger::formatter_glue)
```

Comment:
NIT: i rarely add anything to a script prior to loading package. Is it important to have this prior to loading packages in this case? (Genuinely curious)

Reply:
I don't think this is hugely important, but that's probably because I'm of a mind to not load packages  and replace all the `library()` calls with explicit namespacing (ex: `dplyr::bind_rows()`).

Also, tend to think of these lines as temporary until I can find some time to close up https://github.com/daroczig/logger/issues/73